src/components/burger/types.ts

```diff
@@ -0,0 +1,7 @@
+export interface IBurger {
+  burgerMenuText: {
+    to: string;
+    text: string;
+    icon?: boolean;
+  }[];
+}
```

Comment:
Make typing on the example of Swiper, so clear and can be reused
Example:
export interface IMainSwiper {
  data: Data[];
}

export interface Data {
  src: string;
  href: string;
  title: string;
  subtitle: string;
}